ui/widget.js

```diff
@@ -258,6 +259,7 @@ $.Widget.prototype = {
 		this.bindings = $();
 		this.hoverable = $();
 		this.focusable = $();
+		this.classObject = {};
```

Comment:
If there's one thing you learn from this whole process, it should be to not ask Scott to help with naming!